scripts/mod_loader/modapi/mechedit.lua

```diff
@@ -0,0 +1,509 @@
+local pilot_prev = PILOT_NONE
+
+local function update_mechedit()
+	CUtils.SetMechEditTestHighlightable(true)
+	CUtils.SetMechEditClassHighlightable(true)
+	CUtils.SetMechEditHealthIconHighlightable(true)
+	CUtils.SetMechEditMoveHighlightable(true)
+	CUtils.SetMechEditMoveIconHighlightable(true)
+end
+
+
+function mechedit.GetItemOnCursor()
+	if not Game then
+		return nil
+	end
+	
+	return CUtils.GetMechEditItemOnCursor()
+end
+
+function mechedit.IsTipImage()
+	if not Game then
+		return false
+	end
+	
+	return CUtils.IsMechEditTipImage()
+end
+
+-- Highlightable
+function mechedit.IsTestHighlightable()
+	if not Game then
+		return nil
+	end
+	
+	return CUtils.IsMechEditTestHighlightable()
+end
+
+function mechedit.IsClassHighlightable()
+	if not Game then
+		return nil
+	end
+	
+	return CUtils.IsMechEditClassHighlightable()
+end
+
+function mechedit.IsNameHighlightable()
+	if not Game then
+		return nil
+	end
+	
+	return CUtils.IsMechEditNameHighlightable()
+end
+
+function mechedit.IsPilotNameHighlightable()
+	if not Game then
+		return nil
+	end
+	
+	return CUtils.IsMechEditPilotNameHighlightable()
+end
+
+function mechedit.IsInstallHighlightable()
+	if not Game then
+		return nil
+	end
+	
+	return CUtils.IsMechEditInstallCoreHighlightable()
+end
+
+function mechedit.IsUndoHighlightable()
+	if not Game then
+		return nil
+	end
+	
+	return CUtils.IsMechEditUndoHighlightable()
+end
+
+function mechedit.IsHealthHighlightable()
+	if not Game then
+		return nil
+	end
+	
+	return CUtils.IsMechEditHealthHighlightable()
+end
+
+function mechedit.IsHealthIconHighlightable()
+	if not Game then
+		return nil
+	end
+	
+	return CUtils.IsMechEditHealthIconHighlightable()
+end
+
+function mechedit.IsMoveHighlightable()
+	if not Game then
+		return nil
+	end
+	
+	return CUtils.IsMechEditMoveHighlightable()
+end
+
+function mechedit.IsMoveIconHighlightable()
+	if not Game then
+		return nil
+	end
+	
+	return CUtils.IsMechEditMoveIconHighlightable()
+end
+
+function mechedit.IsInventoryHighlightable()
+	if not Game then
+		return nil
+	end
+	
+	return CUtils.IsMechEditInventoryHighlightable()
+end
+
+function mechedit.IsInventoryUpHighlightable()
+	if not Game then
+		return nil
+	end
+	
+	return CUtils.IsMechEditInventoryUpHighlightable()
+end
+
+function mechedit.IsInventoryDownHighlightable()
+	if not Game then
+		return nil
+	end
+	
+	return CUtils.IsMechEditInventoryDownHighlightable()
+end
+
+function mechedit.IsSkill1Highlightable()
+	if not Game then
+		return nil
+	end
+	
+	return CUtils.IsMechEditSkill1Highlightable()
+end
+
+function mechedit.IsSkill2Highlightable()
+	if not Game then
+		return nil
+	end
+	
+	return CUtils.IsMechEditSkill2Highlightable()
+end
+
+function mechedit.IsAbilityHighlightable()
+	if not Game then
+		return nil
+	end
+	
+	return CUtils.IsMechEditAbilityHighlightable()
+end
+
+-- Hovered
+function mechedit.IsTestHovered()
+	if not Game then
+		return false
+	end
+	
+	return CUtils.IsMechEditTestHovered()
+end
```

Comment:
ℹ Continues returning `true` while in test mech scenario, probably shouldn't?

Reply:
That is an oversight. I will look into this.

Reply:
Fixed in dc1389d.
All `MechEdit.Is[..]Hovered` functions will now always return `false` whenever we are not in the `Region` overview.